python/lsst/ip/isr/isrFunctions.py

```diff
@@ -216,6 +217,113 @@ def growMasks(mask, radius=0, maskNameList=['BAD'], maskValue="BAD"):
         fpSet.setMask(mask, maskValue)
 
 
+def maskITLEdgeBleed(ccdExposure, badAmpDict, itlEdgeBleedSatMinArea=10000,
+                     itlEdgeBleedSatFracLevel=0.8,
+                     itlEdgeBleedModelConstant=0.03,
+                     saturatedMaskName='SAT'):
+    """Mask edge bleeds in ITL detectors.
+
+    Parameters
+    ----------
+    ccdExposure : `lsst.afw.image.Exposure`
+        Exposure to apply masking to.
+    badAmpDict : `dict` [`str`, `bool`]"
+        Dictionary of amplifiers, keyed by name, value is True if
+        amplifier is fully masked.
+    itlEdgeBleedSatMinArea : int
```

Comment:
This should be "\`int\`, optional".  And no quote above on the dict.  Typing the backticks into the github comment box is a challenge.
Similar for the optional ones below.